lib/api_client.js

```diff
@@ -146,14 +146,14 @@ var produceEncryptedDataV2 = function(options, notify) {
 };
 
 APIClient.prototype.promisedEncrypt = function(pt, pw, iter) {
-    if (useWebWorker) {
+    if (useWebWorker && typeof onLoadWorkerLoadAsmCrypto === "function") {
```

Comment:
Unsure if you can see this without the extension, but: "consider adding a /** global: onLoadWorkerLoadAsmCrypto */ comment."